### ISSUE TITLE
Error: access blocked by CORS policy

### ISSUE THREAD

**Issue by Viorae** [parse-community/Parse-SDK-Flutter#704]
### New Issue Checklist

- [ ] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [ ] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [ ] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).
- [ ] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Flutter SDK](https://github.com/parse-community/Parse-SDK-Flutter/releases). 

### Environment

Parse Flutter SDK
- SDK version: `FILL_THIS_OUT`
- Operating system version: `FILL_THIS_OUT`

Server
- Parse Server version: `FILL_THIS_OUT`

**Comment by RodrigoSMarques** [comment]
This error is not package related, but your server configuration : `allowHeaders`.

This is described in the documentation in [pub.dev](https://pub.dev/packages/parse_server_sdk)

```
Early Web support
Currently this requires adding X-Parse-Installation-Id as an allowed header to parse-server. When running directly via docker, set the env var PARSE_SERVER_ALLOW_HEADERS=X-Parse-Installation-Id. When running via express, set [ParseServerOptions](https://parseplatform.org/parse-server/api/master/ParseServerOptions.html) allowHeaders: ['X-Parse-Installation-Id'].

Be aware that for web ParseInstallation does include app name, version or package identifier.
```

See issue:
#490 , #385

**Comment by mtrezza** [comment]
I'm closing this as it does not seem to be a Parse Flutter SDK issue.

- For help with Parse Platform we recommend our [community forum](https://community.parseplatform.org)
- For coding questions we recommend Stack Overflow using the [parse-platform](https://stackoverflow.com/questions/tagged/parse-platform) tag
- For network and server questions we recommend ServerFault using the [parse-server](https://serverfault.com/questions/tagged/parse-server) tag

Feel free to comment if you have any questions and we can re-open this issue.
